lxd/instance/drivers/driver_qemu.go

```diff
@@ -1468,7 +1468,7 @@ func (d *qemu) start(stateful bool, op *operationlock.InstanceOperation) error {
 	}
 
 	// Generate the QEMU configuration.
-	confFile, monHooks, err := d.generateQemuConfigFile(cpuInfo, mountInfo, qemuBus, vsockFD, devConfs, &fdFiles)
+	confFile, monHooks, err := d.generateQemuConfigFile(cpuInfo, mountInfo, qemuBus, unixListener, vsockFD, devConfs, &fdFiles)
```

Comment:
I thought we discussed using an `int` here to pass the fd to replicate the same that is already done for the vsock. Have you tried this?

Reply:
if you mean using `nextVsockID()`, I don't see how I can use it to point qemu to the socket created for viriofsd.

Reply:
I meant passing the fd directly as `int` like it's done for the `vsockFD` variable instead of the listener. This would better fit to the arguments which are currently passed into `generateQemuConfigFile`.

But it's of course not related to the error you are facing now.